deebot_client/commands/xml/charge_state.py

```diff
@@ -27,14 +27,13 @@ def _handle_xml(cls, event_bus: EventBus, xml: Element) -> HandlingResult:
 
```

Comment:
To avoid duplicate code, please remove this function here and add the CargeState to the parent classes:

```python
class GetChargeState(XmlCommandWithMessageHandling, ChargeState):
```

Reply:
XML messages have no "ret" attribute though, how about an ABC that both GetChargeState and ChargeState inherit from so that the charge sub-element parsing stays the same?